src/Symfony/Bundle/FrameworkBundle/Resources/views/Collector/http_client.html.twig

```diff
@@ -0,0 +1,70 @@
+{% extends '@WebProfiler/Profiler/layout.html.twig' %}
+
+{% block toolbar %}
+    {% if collector.traces|length %}
+        {% set icon %}
+            {{ include('@Framework/Icon/http-client.svg') }}
+            {% set status_color = '' %}
+            <span class="sf-toolbar-value">{{ collector.traces|length }}</span>
+        {% endset %}
+
+        {{ include('@WebProfiler/Profiler/toolbar_item.html.twig', { link: profiler_url, status: status_color }) }}
+    {% endif %}
+{% endblock %}
+
+{% block menu %}
+<span class="label {{ collector.traces is empty ? 'disabled' }}">
+    <span class="icon">{{ include('@Framework/Icon/http-client.svg') }}</span>
+    <strong>HTTP Client</strong>
```

Comment:
i propose to show the no. of failed requests (5xx) if any, and change the status color to red. If none, show the no. of  bad requests (4xx) and change the status color to orange, if any.

See logger.html.twig for inspiration :)

Reply:
Nice one! Thanks!